dex/build.sbt

```diff
@@ -54,16 +57,46 @@ inConfig(Compile)(
     ),
     mainClass := discoveredMainClasses.value.headOption,
     run / fork := true
-  ))
+  )
+)
 
 // Docker
 inTask(docker)(
   Seq(
-    additionalFiles ++= Seq(
-      (Universal / stage).value,
-      (Compile / sourceDirectory).value / "container" / "start.sh",
-      (Compile / sourceDirectory).value / "container" / "default.conf"
-    )
+    nameOfImage := "wavesplatform/matcher-server",
+    dockerfile := new Dockerfile {
+
+      val (user, userId)   = ("waves-dex", "113")
+      val (group, groupId) = ("waves-dex", "116")
+
+      val userPath    = s"/var/lib/$user"
+      val sourcesPath = s"/usr/share/$user"
+
+      val entryPointSh = s"$sourcesPath/bin/start-matcher-server.sh"
+
+      from("openjdk:8-jre-slim-buster")
+
+      runRaw(s"""mkdir -p $userPath $sourcesPath $userPath/runtime && \\
+                  |groupadd -g $groupId $group && \\
+                  |useradd -d $userPath -g $groupId -u $userId -s /bin/bash -M $user && \\
+                  |chown -R $userId:$groupId $userPath $sourcesPath && \\
+                  |chmod -R 755 $userPath $sourcesPath && \\
+                  |ln -fs $sourcesPath/log/ var/log/waves-dex""".stripMargin)
```

Comment:
Can we put this to `/var/lib/waves` instead? So the user will see file logs.

Reply:
Done